tests/framework/results/test_context.py

```diff
@@ -70,3 +67,116 @@ def test_add_stratification_raises(
     ctx = ResultsContext()
     with pytest.raises(expected_exception):
         raise ctx.add_stratification(name, sources, categories, mapper, is_vectorized)
+
+
+def _aggregate_state_person_time(self, x: pd.DataFrame) -> float:
+    """Helper aggregator function for observation testing"""
+    return len(x) * (28 / 365.35)
+
+
+@pytest.mark.parametrize(
+    "name, pop_filter, aggregator, additional_stratifications, excluded_stratifications, when",
+    [
+        (
+            "living_person_time",
+            'alive == "alive" and undead == False',
+            _aggregate_state_person_time,
+            [],
+            [],
+            "collect_metrics",
+        ),
+        (
```

Comment:
Why are you doing two pop_filters? Is this stress testing something specific about `add_observation`?

Reply:
It looks like this is just creating two different observers to test.

Reply:
^